app/views/partials/_map_popup.html.erb

```diff
@@ -0,0 +1,26 @@
+<div id="map-popup" class="map-popup card shadow-sm p-2">
```

Comment:
Did you make this file? If yes, what is it good for/why did you add it?

Reply:
Sorry, this was merging problem, I deleted it.